client/network/src/protocol/notifications/behaviour.rs

```diff
@@ -1765,7 +1765,7 @@ impl NetworkBehaviour for Notifications {
 							*c == connection && matches!(s, ConnectionState::Opening))
 						{
 							if !any_open {
-								trace!(target: "sub-libp2p", "External API <= Open({:?})", source);
+								trace!(target: "sub-libp2p", "External API <= Open({}, {})", source, set_id);
```

Comment:
isn't `set_id` printed above at lines 1754-1756?

Reply:
Indeed, and so is the `PeerId`.
I think it doesn't hurt to repeat it, and I suspect that Loki might absorb duplicate identical lines.